### PR TITLE
Feature: ticker API and order_trades API

### LINKED ISSUE #27: feature: ticker API
24h low, 24h high, current price(price of last trade), 24h price change percent, 24h amout, 24h quote amount

similar to kline api

the tricky difference of this API and kline api is, if now is 15:00, ticker API aggregates data from 15:00 yesterday to now, while kline API cares 0:00 UTC yesterday to 0:00 UTC today, 0:00 UTC today to now. The bucket/window is not same.

req:

/restapi/ticker_24h/ETH_USDT

resp example:

```
{
  "market": "ETH_USDT",
  "price_change_percent": "1.96%",
  "last": "1234.56",
  "high": "1300",
  "low": "1200",
  "volume": "8913.30000000",
  "quote_volume": "15.30000000",
  "from": 16xxxx,  // unix timestamp 24h ago
  "to": 16xxx, // now
}
```

### PR DESCRIPTION
**Pull request by noel2004**
Resolve #27 and #49

Use simple, TLS cache in ticker request

Also induce some configuration for the restapi binary

humantime_serde seems to be useful so I induce this crate.

### CODE REVIEW

File: src/restapi/tradingview.rs [hunk 1]
```diff
@@ -71,9 +73,109 @@ pub async fn symbols(req: HttpRequest) -> Result<String, RpcError> {
...
+    log::debug!("query ticker from {} to {}", from_ts, now_ts);
+
+    let ticker_ret: TickerItem = sqlx::query_as(&core_query)
+        .bind(&market_name)
+        .bind(from_ts.naive_utc())
+        .fetch_one(&app_state.db)
+        .await?;
+
+    let ret = TickerResult {
+        market: String::from("ok"),
```
Comment (HAOYUatHZ) [inline src/restapi/tradingview.rs:161]: market_name?

File: src/restapi/tradingview.rs [hunk 2]
```diff
@@ -71,9 +73,109 @@ pub async fn symbols(req: HttpRequest) -> Result<String, RpcError> {
...
+        high: ticker_ret.max.to_f32().unwrap_or(0.0),
+        low: ticker_ret.min.to_f32().unwrap_or(0.0),
+        volume: ticker_ret.sum.to_f32().unwrap_or(0.0),
+        quote_volume: ticker_ret.quote_sum.to_f32().unwrap_or(0.0),
+        from: from_ts.timestamp() as u64,
+        to: now_ts.timestamp() as u64,
+    };
+
+    //update cache
+    may_cached_resp.replace(ret.clone());
```
Comment (HAOYUatHZ) [inline src/restapi/tradingview.rs:176]: 这样是否会不同 market 的被覆盖掉了？

Comment (noel2004) [reply src/restapi/tradingview.rs:176]: omg ... my fault, must fix it

File: src/restapi/tradingview.rs (hunk 1 above)
Comment (noel2004) [reply src/restapi/tradingview.rs]: yes. fixed

**Review by HAOYUatHZ — APPROVED** [review]

**Pull request merged**